v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent INSERT, UPDATE, UPSERT, and DELETE statements within a single transaction. 
+
+To execute statements in parallel, use the `RETURNING NOTHING` clause for the independent SQL statements.
+
+<div id="toc"></div>
+
+## Independent SQL statements
+
+CockroachDB considers SQL statements to be independent if they operate on independent tables and their execution can be safely reordered without affecting their results. 
```

Comment:
@Amruta-Ranade, this is a good start, but I agree with Nathan that it's not clear why this feature is important. You've covered the `how` well, but I don't come away with any sense of `why` you'd want to use this.

Instead of structuring this page the way we do for sql statements, perhaps take a look at some of the pages that cover broader sql features, for example, [Views](view.html) or [Interleaved Tables](interleave-in-parent.html). Even if not the best final structure, it might help you make progress to do what the views page does and divide the page into "Why Use Parallel Statement Execution?" and "How It Works". 

I'm happy to sync in person to sketch out some other ideas as well. @robert-s-lee will have good ideas, too. He should certainly review this at a later stage, since he's recommended this feature to prospects already.